### PR TITLE
modify plugin file

### PR DESCRIPTION
**Pull request by Stevent-fei**
Pull Request Description
---

### Describe what this PR does / why we need it
support new sealer

### Special notes for reviews

### CODE REVIEW

File: build/plugins/preflight_shell_plugin.yaml [hunk 1]
```diff
@@ -4,8 +4,8 @@ metadata:
   name: preflight # Specify this plugin name,will dump in $rootfs/plugin dir.
 spec:
   type: SHELL
-  action: PreInit # PreInit PreInstall PostInstall
-  'on': master0
+  action: post-install # PreInit PreInstall PostInstall
```
Comment (kakaZhou719) [inline build/plugins/preflight_shell_plugin.yaml:7]: pre-init-host

File: build/plugins/install_addons_shell_plugin.yaml [hunk 2]
```diff
@@ -4,8 +4,8 @@ metadata:
   name: install-addons # Specify this plugin name,will dump in $rootfs/plugin dir.
 spec:
   type: SHELL
-  action: PreGuest # PreInit PreInstall PostInstall
-  'on': master0
+  action: pre-install # PreInit PreInstall PostInstall
```
Comment (kakaZhou719) [inline build/plugins/install_addons_shell_plugin.yaml:7]: post-install

File: build/plugins/configure_kubelet_shell_plugin.yaml [hunk 3]
```diff
@@ -4,8 +4,8 @@ metadata:
   name: configure_kubelet # Specify this plugin name,will dump in $rootfs/plugin dir.
 spec:
   type: SHELL
-  action: PreInit|PreJoin # PreInit PreInstall PostInstall
-  'on': master
+  action: pre-install|pre-scaleup # PreInit PreInstall PostInstall
```
Comment (kakaZhou719) [inline build/plugins/configure_kubelet_shell_plugin.yaml:7]: pre-init-host

File: build/plugins/preflight_shell_plugin.yaml (hunk 1 above)
Comment (VinceCui) [reply build/plugins/preflight_shell_plugin.yaml]: pre-init, no need host

**Review by VinceCui — APPROVED** [review]
LGTM

**Pull request merged**